src/main/java/carpetfixes/mixins/reIntroduced/PlayerManager_LlamaRidingDupeMixin_VMPCompat.java

```diff
@@ -0,0 +1,50 @@
+package carpetfixes.mixins.reIntroduced;
+
+import carpetfixes.CFSettings;
+import carpetfixes.helpers.Utils;
+import net.minecraft.entity.Entity;
+import net.minecraft.nbt.NbtCompound;
+import net.minecraft.server.PlayerManager;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Dynamic;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.ModifyArg;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.function.Function;
+
```

Comment:
Same as other pull request add a @Restriction so it only loads if either vmp or C2ME is present